### ISSUE TITLE
Subjects Identification

### ISSUE THREAD

**Issue by GuyTevet** [CMU-Perceptual-Computing-Lab/panoptic-toolbox#14]
Hi,
Thanks again for your great work.

Can you please release identification numbers of the subjects that appears on each sequence?
I'm asking this since I'm intending to create train-test dividing of your data, in which, non of the subjects that appears in the test set are also appearing in the training set.

**Comment by jhugestar** [comment]
Unfortunately, we do not have such label yet. It would be great if you could share it once you could annotate them.
